src/mpol/utils.py

```diff
@@ -10,6 +10,30 @@ def torch2npy(tensor):
     return tensor.detach().cpu().numpy()
 
 
+def center_np_image(image):
```

Comment:
seems like `np.fft.fftshift` would perform this functionality?

Reply:
Yes, my thought was this is better for speed, but then again it wouldn't be in an inference loop anyway, so that doesn't really matter. I'll remove it.